uportal-war/src/main/webapp/media/skins/respondr/common/bootstrap/variables.less

```diff
@@ -14,7 +14,7 @@
 @gray-light:             lighten(@gray-base, 46.7%); // #777
 @gray-lighter:           lighten(@gray-base, 93.5%); // #eee
 
-@brand-primary:         darken(#428bca, 6.5%); // #337ab7
+@brand-primary:         darken(#008bca, 6.5%); // #337ab7
```

Comment:
I made a mistake in changing this. As far as I can tell it was only the uPortal colors (color3 and color3-light) that were actually causing issues. Bootstrap's color wasn't, and I changed it back.